config/routes.rb

```diff
@@ -111,6 +111,8 @@
   match 'movies/preview'
   match 'movies/snapshot'
 
+  match 'plugin/user'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.